uvicorn/middleware/proxy_headers.py

```diff
@@ -68,7 +68,8 @@ class _TrustedHosts:
     """Container for trusted hosts and networks"""
 
     def __init__(self, trusted_hosts: list[str] | str) -> None:
-        self.always_trust: bool = trusted_hosts == "*"
+        # gunicorn sends trusted hosts as a parsed list of strings while uvicorn forwards as is provided by input
```

Comment:
I don't want references to `gunicorn` anymore.
```suggestion
```
This change makes sense on the uvicorn context as well.

Reply:
Got that. But just to clarify on uvicorn context, uvicorn does not send trusted hosts as `["*"]` to `_TrustedHosts` in both cases as a Command line program or as an explicit application parameter.

So that's why I thought that gunicorn is the only one who is sending that value after passing it in a lits.